schema-manager/src/main.rs

```diff
@@ -223,27 +235,27 @@ fn run(log: &Logger) -> Result<(), Box<dyn std::error::Error>>
     let boray_config = get_boray_config();
     let boray_port = boray_config.server.port;
 
-    let fast_arg = [String::from("tcp://"), boray_host, String::from(":"), boray_port.to_string()]
-                    .concat();
+    let fast_arg = [
+        String::from("tcp://"),
+        boray_host,
+        String::from(":"),
+        boray_port.to_string(),
+    ]
+    .concat();
     info!(log, "pnode argument to electric-boray:{}", fast_arg);
-    let eb_endpoint = [eb_address, String::from(":"), DEFAULT_EB_PORT.to_string()]
-        .concat();
+    let eb_endpoint = [eb_address, String::from(":"), DEFAULT_EB_PORT.to_string()].concat();
     info!(log, "electric-boray endpoint:{}", eb_endpoint);
     let mut stream = TcpStream::connect(&eb_endpoint).unwrap_or_else(|e| {
         error!(log, "failed to connect to electric-boray: {}", e);
         process::exit(1)
     });
 
     let mut msg_id = FastMessageId::new();
-    let recv_cb = |msg: &FastMessage| { vnode_response_handler(&log, msg) };
+    let recv_cb = |msg: &FastMessage| vnode_response_handler(&log, msg);
```

Comment:
Not sure about this change to the closure, I believe I tried it this way and it didn't work. I will test it out.

Reply:
Changes on schema-manager look good:
```
[root@66f23311 (boray) /opt/smartdc/boray/bin]$ ./schema-manager
INFO: Running "/usr/sbin/mdata-get SAPI_URL" ...
Nov 26 18:31:30.944 INFO sapi_url:http://sapi.jonran-dc.joyent.us, build-id: 0.1.0
INFO: Running "/usr/bin/zonename" ...
Nov 26 18:31:31.196 INFO pnode argument to electric-boray:tcp://1.boray.jonran-region.joyent.us:2030, build-id: 0.1.0
Nov 26 18:31:31.196 INFO electric-boray endpoint:electric-boray.jonran-region.joyent.us:2020, build-id: 0.1.0
Nov 26 18:31:31.225 INFO processing vnode: "0", build-id: 0.1.0
Nov 26 18:31:31.228 INFO Creating boray user and role if they don't exist on postgres://postgres:postgres@10.77.77.22:5432, build-id: 0.1.0
Nov 26 18:31:31.263 INFO Creating boray database if it doesn't exist on postgres://postgres:postgres@10.77.77.22:5432, build-id: 0.1.0
Nov 26 18:31:31.305 INFO Creating boray schemas on postgres://boray:boray@10.77.77.22:5432/boray, build-id: 0.1.0
Nov 26 18:31:31.310 INFO processing vnode: "2", build-id: 0.1.0
Nov 26 18:31:31.310 INFO Creating boray user and role if they don't exist on postgres://postgres:postgres@10.77.77.22:5432, build-id: 0.1.0
Nov 26 18:31:31.329 INFO Creating boray database if it doesn't exist on postgres://postgres:postgres@10.77.77.22:5432, build-id: 0.1.0
Nov 26 18:31:31.432 INFO Creating boray schemas on postgres://boray:boray@10.77.77.22:5432/boray, build-id: 0.1.0
Nov 26 18:31:31.436 INFO Done., build-id: 0.1.0
```